fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -107,13 +108,26 @@
     String importPartitionResourceType;
 
 
+    /**
+     * If validate FHIR resources.
+     */
+    @Inject
+    @BatchProperty(name = Constants.IMPORT_FHIR_IS_VALIDATION_ON)
+    String fhirValidation;
+
+
     public ChunkWriter() {
         super();
     }
 
+    // This is for the warning triggered by IMPORT_IS_COLLECT_OPERATIONOUTCOMES which controls if upload OperationOutcomes to COS/S3.
+    @SuppressWarnings("unused")
```

Comment:
this is used

Reply:
<img width="978" alt="Screen Shot 2020-03-18 at 12 44 03 PM" src="https://user-images.githubusercontent.com/54077144/76985174-30f85700-6916-11ea-8abd-0eba202bea03.png">

Reply:
if remove the suppress, we will get this ^^^^^

Reply:
I think that's the right behavior to display. 
by hiding it, we'll not get the warnings. (I suggest flipping the expressions, check that it's not null, and then the constant)

Reply:
OK, I can remove the suppress